Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.h

```diff
@@ -26,11 +26,14 @@
 #include <gst/video/video-format.h>
 #include <gst/video/video-info.h>
 #include <wtf/MediaTime.h>
+#include <wtf/Seconds.h>
 
 namespace WebCore {
 
 using GstEventSeqNum = uint32_t;
 class IntSize;
+// NOTE: YouTube 2018 has tests that expect this to be >= 5s.
+const WTF::Seconds GST_EME_LICENSE_KEY_RESPONSE_TIMEOUT = WTF::Seconds(6);
```

Comment:
I would add this to a GStreamerEMECommon.h file because it belongs only to EME. That file should be properly flagged.

I won't oppose to have it here, but I'd request then you flag this for ENCRYPTED_MEDIA